SWLOR.Game.Server/Feature/AbilityDefinition/Force/DisturbanceAbilityDefinition.cs

```diff
@@ -25,15 +26,51 @@ public class DisturbanceAbilityDefinition : IAbilityListDefinition
             return _builder.Build();
         }
 
-        private void Impact(uint activator, uint target, int dmg, int accDecrease, int tier, string effectTag, int dc)
+        private static void ImpactAction(uint activator, uint target, int level, Location targetLocation)
         {
+            var willBonus = GetAbilityScore(activator, AbilityType.Willpower);
+            var dmg = 0;
+            string effectTag = "";
+            int dc = 0;
```

Comment:
Always prefer "var" instead of specific types. Please replace these to var for consistency.